UPGRADING.md

```diff
@@ -203,3 +203,13 @@ const App = () => {
   );
 };
 ```
+
+### 6.x -> 7.x
+
+- `ActiveAccountContextProvider` and `ActiveProjectContextProvider` will now
+  _not_ render their children until the active account + project have been
+  resolved. If there is no active account or project, the `InviteRequired`
+  screen will be rendered in the place of children.
+
+- The `useActiveProject` hook now returns non-optional data. You can simplify
```

Comment:
maybe worth adding a small example to illustrate what we mean by this

```diff
--- a/use.ts
+++ b/use.ts
-      project?.id
+      project.id



```

Reply:
@jkdowdle Great idea. Unless there are other changes, I'll add that in a follow-up.